### PR TITLE
Renaming the environment variable for analytics

### PR DESCRIPTION
**Pull request by bcdurak**
## List of Changes

- I have renamed the environment variable from "MLSTACKS_ANALYTICS_OPT_OUT" to "MLSTACKS_ANALYTICS_OPT_IN" and made the necessary adjustments.

### CODE REVIEW

File: src/mlstacks/constants.py [hunk 1]
```diff
@@ -11,6 +11,29 @@
...
+    """
+    value = os.getenv(var)
+
+    if value in ["1", "y", "yes", "True", "true"]:
+        return True
+
+    if value in ["0", "n", "no", "False", "false"]:
+        return False
+
+    return default
```
Comment (strickvl) [inline src/mlstacks/constants.py:35]: Should this be a util function instead of living in constants.py?

**Review by strickvl — APPROVED** [review]
Aside from the question, this looks good from my side!

File: src/mlstacks/constants.py (hunk 1 above)
Comment (bcdurak) [reply src/mlstacks/constants.py:35]: I copied the behavior from the main ZenML repo. If you think that it is more convenient, we can move it to a file like: `utils/environment_utils.py`.

Comment (strickvl) [reply src/mlstacks/constants.py:35]: Yes I think that'd be better.

Comment (bcdurak) [reply src/mlstacks/constants.py:35]: On it!

**Pull request merged**